src/lib/assets/yaml.ts

```diff
@@ -77,7 +79,9 @@ export async function overridesFile({ hash, name, image, config, apiToken }: Ass
       env: [
         { name: "PEPR_WATCH_MODE", value: "true" },
         { name: "PEPR_PRETTY_LOG", value: "false" },
-        { name: "LOG_LEVEL", value: "debug" },
+        { name: "LOG_LEVEL", value: "info" },
+        process.env.PEPR_MODE === "dev" && { name: "MY_CUSTOM_VAR", value: "example-value" },
+        process.env.PEPR_MODE === "dev" && { name: "ZARF_VAR", value: "###ZARF_VAR_THING###" },
```

Comment:
```suggestion
        process.env.PEPR_MODE === "dev" && { name: "MY_CUSTOM_VAR", value: "example-value" },
        process.env.PEPR_MODE === "dev" && { name: "ZARF_VAR", value: "###ZARF_VAR_THING###" },
```

Can we please remove these lines? We cover this below this variable by pushing into the env array.